client/my-sites/earn/ads/payments.jsx

```diff
@@ -101,21 +78,19 @@ class WordAdsPayments extends Component {
 		);
 	}
 
-	notices( payments, wordAdsSettings ) {
```

Comment:
Updated param names because lint errors were complaining about using the same variable names (ie, payments in parent scope vs payments in function scope).